lib/cms/helpers.ex

```diff
@@ -69,6 +69,22 @@ defmodule CMS.Helpers do
     |> Enum.map(&File.from_api/1)
   end
 
+  @spec parse_page_types(map) :: list(String.t())
+  def parse_page_types(%{} = data) do
+    data
+    |> Map.get("field_page_type", [])
+    |> Enum.map(&Map.get(&1, "name"))
+    |> Enum.filter(&(!is_nil(&1)))
+  end
+
+  @spec parse_related_transit(map) :: list(String.t())
+  def parse_related_transit(%{} = data) do
+    data
+    |> Map.get("field_related_transit", [])
+    |> Enum.map(&Map.get(&1, "name"))
+    |> Enum.filter(&(!is_nil(&1)))
```

Comment:
suggestion: use `Enum.reject/2` instead of `Enum.filter/2`.

This code may also be made clearer using the `Access` module for nested traversal instead of mapping `Map.get`:

```suggestion
    data
    |> Map.get("field_related_transit", [])
    |> get_in([Access.all(), "name"])
    |> Enum.reject(&is_nil(&1))
```

it's possible to combine the first `Map.get` in the call to `get_in` and use `Access` for that too, but it would require checking if the result was nil before calling `reject`/`filter` because it doesn't allow us to specify a default value of `[]` as we do here so it's probably a wash.